### DISCUSSION TITLE
iniciando el prep henry

### DISCUSSION THREAD

**Discussion by Franks1985** [atralice/Curso.Prep.Henry#94] [Q&A]
hola, tengo dudas con respecto a mi homework, recien he comenzado ayer a realizar el curso, estoy en la homework 02-JS-I, estoy siguiendo las instrucciones del video pero al pasar el test me dice error, por favor alguien que me pueda orientar, gracias de antemano.
adjunto el error...

Jaklins@DESKTOP-P1JA51M MINGW64 ~/Desktop/henry prep/Curso.Prep.Henry/02-JS-I/homework (master)
$ npm test JSI.test.js

> prep@1.0.0 test C:\Users\Jaklins\Desktop\henry prep\Curso.Prep.Henry
> jest --collectCoverage=false --testResultsProcessor ./.reporter/index.js --detectOpenHandles "JSI.test.js"

● Validation Error:

  Module ./.reporter/index.js in the testResultsProcessor option was not found.
         <rootDir> is: C:\Users\Jaklins\Desktop\henry prep\Curso.Prep.Henry

  Configuration Documentation:
  https://jestjs.io/docs/configuration

npm ERR! Test failed.  See above for more details.

**Comment by alphaier96** [comment]
Hola, lo pudiste solucionar ? tengo el mismo problema.

**Comment by daniel-monsalve-villegas** [comment]
Hola, debes instalar el jest desde npm, "npm i jest"
